apps/ledger-live-mobile/src/screens/Nft/NftInfoNotAvailable.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Link, Text } from "@ledgerhq/native-ui";
+import React from "react";
+import { Trans } from "react-i18next";
+import { Linking } from "react-native";
+import { urls } from "../../config/urls";
+
+export const notAvailableModalInfo = [
+  {
+    title: <Trans i18nKey={"nft.modalDisabled.title"} />,
+    description: (
+      <>
+        <Box mb={3}>
+          <Text fontWeight="medium" mb={2}>
+            <Trans i18nKey={"nft.modalDisabled.body"} />
+          </Text>
+        </Box>
+        <Link
+          type="color"
+          size="medium"
+          onPress={() => Linking.openURL(urls.ratingsContact)} // TODO REPLACE WHEN SUPPORT LINK IS AVAILABLE
```

Comment:
Just a reminder to update this with the right link, cause I guess `ratingsContact` isn't going to help much 😝